examples/creatives/hover_nonlinear.js

```diff
@@ -0,0 +1,89 @@
+const Y_OFFSET_PERCENTAGE = 8;
+const X_OFFSET_PERCENTAGE = 8;
+const WIDTH_PERCENTAGE = 1.1;
+const HEIGHT_PERCENTAGE = 2;
+
+class HoverNonLinear extends BaseSimidCreative {
+    constructor() {
+        super();
+
+        this.initialDimensions_ = null;
+    }
+
+    /** @override */
+    onInit(eventData) {
+        super.onInit(eventData);
+        this.initialDimensions_ = this.environmentData.creativeDimensions;
+    }
+
+    /** @override */
+    onStart(eventData) {
+        super.onStart(eventData);
+        this.addActions_();
+    }
+
+    /**
+     * Adds actions to different buttons available on the overlay.
+     * @private
+     */
+    addActions_() {
+        this.sendMessageOnClick_("close_ad", CreativeMessage.REQUEST_STOP);
```

Comment:
use single quotes everywhere instead of double quotes.

Reply:
okay, will do